app/routes/admin/delete/dogs.js

```diff
@@ -24,7 +17,7 @@ module.exports = [
     options: {
       auth: { scope: [admin] },
       handler: async (request, h) => {
-        const sort = { column: 'status', order: 'ASC' }
+        const sort = { column: request.query.sortKey ?? 'status', order: request.query.sortOrder ?? 'ASC' }
 
```

Comment:
are you able to put a query schema check here - anything could be added as sortKey.